lib/screens/onboarding/onboarding.dart

```diff
@@ -7,30 +7,39 @@ import 'package:seeds/screens/onboarding/onboarding_view_model.dart';
 
 class Onboarding extends StatelessWidget {
   final List<PageViewModel> featurePages = [
-    OnboardingViewModel(
+    OnboardingViewModel.rich(
       bubble: null,
       mainImage: 'assets/images/onboarding1.png',
-      body:
-        "Make global payments with zero fees. Earn rewards when you support 'regenerative' organisations and people.",
+      body: [
+        TextSpan(
+          text: "Make payments globally without any fees. Earn rewards when you suport 'regenerative' organisations and people.",
```

Comment:
Spelling error: support

Reply:
lol... can't believe this... i got this copy-pasted...